app/directors-corner.tsx

```diff
@@ -1,5 +1,91 @@
-import WorkInProgress from '@/components/work-in-progress';
+import { DM_Sans, DM_Serif_Display } from 'next/font/google';
+import Image from 'next/image';
+
+import HorsesRunning from '@/components/horses-running';
+
+const dmSerifDisplay = DM_Serif_Display({
+  weight: '400',
+  subsets: ['latin'],
+  display: 'swap',
+});
+
+const dmSans = DM_Sans({
+  weight: '400',
+  subsets: ['latin'],
+  display: 'swap',
+});
```

Comment:
Remove these since we have the styles globally.

Reply:
Fixed